include/sisl/fds/buffer.hpp

```diff
@@ -49,7 +49,7 @@ struct sg_list {
 struct sg_iterator {
     sg_iterator(const sg_iovs_t& v) : m_input_iovs{v} { assert(v.size() > 0); }
 
-    sg_iovs_t next_iovs(uint32_t size) {
+    sg_iovs_t next_iovs(uint32_t const size) {
```

Comment:
This is a backwards incompatible-change to the api. Is it needed?

Reply:
The const qualifier makes sense as the param is size. I overlooked the fact that this makes it backwards incompatible. This addition should be fine once I bump the major version correct?